pkg/database/database.go

```diff
@@ -32,18 +35,33 @@ type Database struct {
 	resourceTableName string
 }
 
-func NewDatabase() (*Database, error) {
+func GetDatabaseConnection() (*Database, error) {
```

Comment:
I am not sure about this change of name, I've usually seen that in go the "constructors" are called New<StructName> in this case, this function instantiates a new instance of the struct Database declared above, so, I would keep the name `NewDatabase()` instead of `GetDatabaseConnection()`

Plus, the name `GetDatabaseConnection` suggests me that we are retrieving a DB connection that has been already established, and that is not the case, we intend to create the sql.DB instance once (as it is a connection pool) and keep it until the program ends.

Reply:
I saw  our code and we are using the same convention in all the places. So, I will change then the name of this function to the old one.